seqr/views/utils/dataset_utils.py

```diff
@@ -366,44 +350,59 @@ def _load_rna_seq_file(
     missing_required_fields = defaultdict(list)
     gene_ids = set()
     current_sample = None
-    for line in tqdm(f, unit=' rows'):
-        row = dict(zip(header, _parse_tsv_row(line)))
-        for sample_id, row_dict in _parse_rna_row(
-                row, column_map, required_column_map, missing_required_fields, allow_missing_gene, **kwargs):
-            tissue_type = TISSUE_TYPE_MAP[row[TISSUE_COL]]
-            project = row[PROJECT_COL]
-            sample_key = (sample_id, project, tissue_type)
-
-            if sample_key in potential_loaded_samples:
-                loaded_samples.add(sample_key)
-                continue
+    for line in tqdm(parsed_f, unit=' rows'):
+        row = dict(zip(header, line))
```

Comment:
I created `parsed_f` in my last PR but forgot to actually use it in the iteration. ops